stable/redis-ha/templates/redis-slave-appliance-vpc.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: redis-slave-appliance-vpc
+  annotations:
+    cloud.google.com/load-balancer-type: "Internal"
+spec:
+  ports:
+  - port: 6379
+    protocol: TCP
+    targetPort: 6379
+  selector:
+    app: "redis-ha"
+    redis-node: "true"
+    redis-role: "slave"
+  type: LoadBalancer
```

Comment:
There is a reason to use a `LoadBalancer` service type ?

Reply:
Hey Ismail. Hey From a high level, there are only two types of pods now: sentinels and servers. As before, they both use the same docker image. Sentinels are still looking for an environment variable to tell them to specialize. During initial launch, the server pods each perform a deterministic check to determine which of them should become the master. Once the master is up, the sentinels connect to it and they assign roles to to remaining slaves.

The sentinels move servers in/out of the master/replica LBs as needed using the new promote.sh script. This way individual redis servers can fail and be replaced without data loss or downtime beyond what it took to reach +odown. 

Multiple parallel executions of the chart in the same K8S cluster are explicitly supported. There are a few new hooks to make customizing and debug a bit easier. Labels such as podID and runID are set for each server on launch. Server promotion and initial label setting have their own scripts.